Assets/MixedRealityToolkit/Interfaces/Services/IMixedRealityDataProvider.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.MixedReality.Toolkit
     /// <summary>
     /// Generic interface for all Mixed Reality Data providers
```

Comment:
>Generic interface for all Mixed Reality Data providers [](start = 8, length = 54)

A brief explanation here on when a type should be a data provider instead of just a service could be useful.

Reply:
Good suggestion. Will add that in the near future.